frontend/src/scenes/billing/BillingCTAHero.tsx

```diff
@@ -0,0 +1,64 @@
+import { LemonButton } from '@posthog/lemon-ui'
+import { useActions, useValues } from 'kea'
+import { BlushingHog } from 'lib/components/hedgehogs'
+import useResizeObserver from 'use-resize-observer'
+
+import { BillingProductV2Type } from '~/types'
+
+import { billingLogic } from './billingLogic'
+import { billingProductLogic } from './billingProductLogic'
+import { PlanComparisonModal } from './PlanComparison'
+
+export const BillingCTAHero = ({ product }: { product: BillingProductV2Type }): JSX.Element => {
+    const { width, ref: billingHeroRef } = useResizeObserver()
+
+    const { redirectPath } = useValues(billingLogic)
+    const { isPlanComparisonModalOpen, billingProductLoading } = useValues(billingProductLogic({ product }))
+    const { toggleIsPlanComparisonModalOpen, setBillingProductLoading } = useActions(billingProductLogic({ product }))
+
+    // TODO(@zach): add multiple variations of this copy
+    return (
+        <div className="flex relative justify-between items-center rounded-lg bg-mark" ref={billingHeroRef}>
+            <div className="p-4">
+                {/* <h1>Add your credit card to unlock all the features.</h1> */}
+                <h1>Upgrade to unlock all the features.</h1>
+                <h1 className="text-danger">Only pay for what you use.</h1>
+                <p className="mt-2 mb-0">
+                    You're currently on the free plan. It's free but limited in features. Add your credit card to
+                    upgrade to our paid plan where you pay per use (after the generous free tier).
+                </p>
```

Comment:
I've pushed some changes to the text here

<img width="882" alt="Arc 2024-06-17 19 16 10" src="https://github.com/PostHog/posthog/assets/18598166/6da9545e-08b1-41cd-aa8b-f2f17ae04991">